core/templates/dev/head/pages/collection_editor/collection_editor_navbar_directive.html

```diff
@@ -96,7 +96,7 @@
       </button>
 
       <button type="button" ng-click="unpublishCollection()"
-              ng-if="!isPrivate && canUnpublish"
+              ng-if="!collectionRightsObject.isPrivate && collectionRightsObject.canUnpublish()"
```

Comment:
Shouldn't this be ``!collectionRightsObject.isPrivate()``? Didn't this produce a console error when you were testing it?